Source/ui_qt/DebugSupport/RegViewVU.cpp

```diff
@@ -0,0 +1,189 @@
+#include <stdio.h>
+#include <QHeaderView>
+#include <QMenu>
+#include "RegViewVU.h"
+#include "string_format.h"
+
+#define CLIP_FLAG_MASK 0xFFFFFF
+
+CRegViewVU::CRegViewVU(QWidget* parent, CMIPS* ctx)
+    : CRegViewPage(parent)
+    , m_ctx(ctx)
+{
+	AllocateTableEntries(2, 45 + 16);
+	setColumnWidth(0, 46);
+	horizontalHeader()->setStretchLastSection(true);
+	for(unsigned int x = 0; x < 32; x++)
+	{
+		if(x < 10)
+			WriteTableLabel(x, "VF0%i", x);
+		else
+			WriteTableLabel(x, "VF%i", x);
+	}
+	WriteTableLabel(32, "ACC");
+	WriteTableLabel(33, "Q");
+	WriteTableLabel(34, "I");
+	WriteTableLabel(35, "P");
+	WriteTableLabel(36, "R");
+	WriteTableLabel(37, "MACF");
+	WriteTableLabel(38, "STKF");
+	WriteTableLabel(39, "CLIP");
+	WriteTableLabel(40, "PIPE");
+	WriteTableLabel(41, "PIPEQ");
+	WriteTableLabel(42, "PIPEP");
+	WriteTableLabel(43, "PIPEM");
+	WriteTableLabel(44, "PIPEC");
+	for(unsigned int x = 0; x < 16; x++)
+	{
+		if(x < 10)
```

Comment:
Wouldn't it be possible to use a better formatting string to avoid a `if` here? I think `%02d` would do the trick.